cli/command/stack/kubernetes/convert.go

```diff
@@ -19,14 +19,23 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// PullSecretExtraField is an extra field on ServiceConfigs usable to reference a pull secret
+	PullSecretExtraField = "x-pull-secret"
```

Comment:
Looks like these consts can be un-exported?

Reply:
They might be used in Compose on Kubernetes v1beta1 handler at some point (to enable their usage from UCP UI)

Reply:
"at some point"

generally it's better to change when needed, and not before

Reply:
alright, I'll change it :)